substrate/primitives/api/proc-macro/src/decl_runtime_apis.rs

```diff
@@ -528,6 +532,7 @@ fn generate_runtime_info_impl(trait_: &ItemTrait, version: u32) -> TokenStream {
 
 	quote!(
 		#crate_::std_enabled! {
+			#[allow(deprecated)]
```

Comment:
can't we also use retrieve and propagate the `allow(deprecated)` attributes for this element?

Reply:
now uses maybe_allow_attrs for the `TraitItem` in question